go.mod

```diff
@@ -1,7 +1,7 @@
 module github.com/terraform-providers/terraform-provider-datadog
 
 require (
-	github.com/DataDog/datadog-api-client-go/v2 v2.7.1-0.20230120075132-4b9844ff6888
+	github.com/DataDog/datadog-api-client-go/v2 v2.7.1-0.20230201102337-c1bc90fae237
```

Comment:
Note: https://github.com/DataDog/datadog-api-client-go/pull/1843 has just been merged. I will update this to point to the latest version of datadog-api-client-go once a new release is published next week